src/bp-groups/bp-groups-functions.php

```diff
@@ -3766,3 +3766,55 @@ function groups_can_user_manage_messages( $user_id, $group_id ) {
 
 	return $is_allowed;
 }
+
+function bp_groups_prime_mentions_results() {
```

Comment:
@mehulkaklotar It would be great we can add doc block.

Reply:
done.